src/DataFrames.jl

```diff
@@ -9,14 +9,23 @@ module DataFrames
 
 using Reexport, StatsBase, SortingAlgorithms, Compat
 @reexport using CategoricalArrays, Missings
-using Base: Sort, Order
+using Base.Sort
+using Base.Order
 using Compat.Unicode
 using Compat.Printf
 
 if VERSION >= v"0.7.0-DEV.2738"
+    using Unicode: normalize
+
     const kwpairs = pairs
 else
     kwpairs(x::AbstractArray) = (first(v) => last(v) for v in x)
+    macro isdefined(sym)
```

Comment:
Would be nice to contribute this to Compat if you have the time.

Reply:
I did that a while back but it was rejected

Reply:
yeah, as far as I know `@isdefined` has been around for quite a while, so I think there's some reason it's not in `Compat`, though I have no idea what that is.  @ararslan , do you think you could find that issue or thread?

Reply:
https://github.com/JuliaLang/Compat.jl/pull/402

Reply:
ok, can we perhaps keep it in DataFrames though?  I'm trying very hard to put *only* 0.7 valid code into the main body so that we don't really have to do anything once we no longer support 0.6 (which considering the nature of 0.7 may not be very long)

Reply:
OK, let's discuss this in Compat. But here I think you can just do `isdefined(Base, :unique!)`, the fact that `Base` was omitted is most likely an oversight.